telemetry/definitions/commonDefinitions.json

```diff
@@ -196,6 +201,11 @@
             "description": "Called when a stream is searched",
             "metadata": [{"type": "result"}]
         },
+        {
+            "name": "cloudwatchlogs_searchGroup",
```

Comment:
this doesn't apply for speedsearch right? seems a bit too noisy if it does.

Reply:
Nope, only for the search box that actually queries the service